app/assets/javascripts/components/page/page_overlay.es6.jsx

```diff
@@ -88,9 +90,24 @@ class PageOverlay extends Component {
     }
   }
 
-  // --------------------------------------------------
-  // Render
-  // --------------------------------------------------
+//  --------------------------------------------------
+//  Render
+//  --------------------------------------------------
+  renderModal() {
+    if (this.props.overlay.type === 'edit') {
+      return (
+        <EditModal
+          overlay={this.props.overlay}
+          student={this.props.student} />
+      );
+    }
```

Comment:
Can we add an `else` clause here (I know it's not needed but it's a bit more explicit).